proto/decentraland/sdk/components/tween.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package decentraland.sdk.components;
+
+import "decentraland/sdk/components/common/id.proto";
+
+option (common.ecs_component_id) = 1101;
+
+// The Tween component allows you to animate the final transform of an entity.
+// The animation is performed time-traveling back in time .duration amount of milliseconds
+// and interpolating the values of the transform from the past.
+//
+// The parameters used for this time-travel are set by the transform component and its
+// modifiers like billboard, avatar attachment point, etc.
+//
+// When an entity changes its parent entity, the tween's time-travel registry will
+// automatically reset. Meaning NO tweening will be performed upon reparenting.
+//
+// Tweens will be calculated AFTER the final position of the entity is calculated,
+// and BEFORE the position of the child entities is computed. Effectively applying
+// the animation to all its children.
+//
+// It is important to mention that tweens only apply to the local entity, if a parent entity P
+// has a linear tween, and it has a child entity C with another tween, by moving P, only P's tween will apply,
+// moving C's frame of reference but NOT animating it.
```

Comment:
This is super interesting. I found it hard to grasp looking at text, though. A link to a spec with images (or even GIFs) would be _awesome_.

Let's not, though 😂  sounds like a ton of work that nobody will benefit from for the foreseeable future.